### PR TITLE
HCI and Flow Control logging

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

File: subsys/bluetooth/controller/hci/hci.c [hunk 1]
```diff
@@ -257,6 +257,7 @@ static void host_buffer_size(struct net_buf *buf, struct net_buf **evt)
 		return;
 	}
 
+	BT_INFO("FC: host buf size: %d", acl_pkts);
```
Comment (cvinayak) [inline subsys/bluetooth/controller/hci/hci.c]: All these should be BT_DBG, right?

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/hci/hci.c (hunk 1 above)
Comment (carlescufi) [inline subsys/bluetooth/controller/hci/hci.c]: No, I think INFO is fine here. It's informational for the observer

Comment (jhedberg) [inline subsys/bluetooth/controller/hci/hci.c]: BT_INFO should be used very conservatively. It's basically for information that can be considered useful "almost always" regardless of what the user is testing or debugging. I don't think this qualifies. BT_INFO gets enabled whenever CONFIG_BLUETOOTH_DEBUG is enabled, whereas BT_DBG gets enabled when debug has been enabled for the specific module in question.

**Review by jhedberg — CHANGES_REQUESTED** [review]
Looks fine, except for the BT_INFO vs BT_DBG thing. Even without debug logs enabled you'll still be able to see this if you've enabled HCI logging.

**Comment by carlescufi** [comment]
Pushed an update

**Review by jhedberg — APPROVED** [review]

**Pull request merged**